2698. Find the Punishment Number of an Integer/solution.py

```diff
@@ -0,0 +1,71 @@
+class Solution:
+    # Given a positive integer n, return the punishment number of n which
+    # is determined by adding up the squares of any integer i that is 
+    # between 1 and n (inclusive) and the following complicated condition:
+    # The square of i can be partitioned into contiguous substrings so that
+    # the sum of the integers values of the substrings is equal to i.
+    def punishmentNumber(self, n: int) -> int:
+
```

Comment:
nit: extra whitespace

Reply:
@winry-fruitiwi - Where's the extra white space?